README.md

```diff
@@ -5,19 +5,26 @@
 [![Go Report Card](https://goreportcard.com/badge/github.com/jenkinsci/kubernetes-operator "Go Report Card")](https://goreportcard.com/report/github.com/jenkinsci/kubernetes-operator)
 [![Docker Pulls](https://img.shields.io/docker/pulls/virtuslab/jenkins-operator.svg)](https://hub.docker.com/r/virtuslab/jenkins-operator/tags)
 
-Go to [**our documentation website**](https://jenkinsci.github.io/kubernetes-operator/) for more information, with examples and guides.
-
 ![logo](/assets/jenkins_gopher_wide.png)
 
-## What's the Jenkins Operator?
+## What's the Jenkins Operator ?
```

Comment:
there's a ` ` [space] before `?` that should not be there

Reply:
I ahve updated this :)